rcl/test/rcl/test_graph.cpp

```diff
@@ -44,6 +44,9 @@
 # define CLASSNAME(NAME, SUFFIX) NAME
 #endif
 
+bool is_connext =
+  std::string(rmw_get_implementation_identifier()).find("connext") != std::string::npos;
```

Comment:
Is this `find` specific enough or should it better check for `find("rmw_connext") == 0`?